src/views/Error.tsx

```diff
@@ -0,0 +1,10 @@
+import { View, Text } from 'react-native';
+import type { RouterProps } from '../types/routerTypes';
+
+export function Error(_: RouterProps) {
```

Comment:
Is `_: RouterProps` bit needed given that it is not used, or is this a placeholder for future?